api-lib/gql/__generated__/zeus/const.ts

```diff
@@ -287,6 +288,7 @@ export const AllTypesProps: Record<string, any> = {
     hash: 'String_comparison_exp',
     name: 'String_comparison_exp',
     read_circle: 'Boolean_comparison_exp',
+    read_discord: 'Boolean_comparison_exp',
```

Comment:
@levity these seem like erroneous additions to the types... ? I don't see any reference to `read_discord` anywhere else in the codebase, and a local `yarn generate` on my machine removes these additions as well. Are these intentional? Do you perhaps have an extra column in your local db that is generating these?